test/app-luatest/gh_7762_luajit_enable_checkhook_test.lua

```diff
@@ -0,0 +1,69 @@
+local checktrace = require('jit.util').traceinfo
+local t = require('luatest')
+local g = t.group()
+
+-- Test payload to be recorded. It calculates the arithmetic sum
+-- of elements from 1 to the given n.
+local function sumN(n)
+    local result = 0
+    for i = 1, n do
+        result = result + i
+    end
+    return result
+end
+
+g.test_luajit_enable_checkhook = function()
+    -- Setup JIT engine to record a trace as soon as possible.
+    jit.opt.start('hotloop=1')
+    -- Purge all existing traces.
+    jit.flush()
+
+    -- Check there is no traces at the beginning of the test.
+    assert(not checktrace(1), "JIT flush doesn't work")
```

Comment:
Minor: we may mention, that `checktrace()` is a LuaC function, so no traces will be compiled for it here.

Reply:
I believe this is quite clear for ones who is concerned about this fact, and I doubt this particular comment you're suggesting above make something clear to the others.

Ignoring.